desktop/conf/pseudo-distributed.ini.tmpl

```diff
@@ -816,6 +816,14 @@
    ## collection_interval=30000
 
```

Comment:
For info there is also a similar config file to update https://github.com/cloudera/hue/blob/master/desktop/conf.dist/hue.ini
(the production one)

Reply:
Oh, will update it too. Thanks @romainr !